### PR TITLE
Standardize error messages

### PR DESCRIPTION
**Pull request by jamessinmaojun**
According to the [TP Feature Testing Document](https://docs.google.com/document/d/1hq0cifK01_vj3jizP8J8WDTIrOc--FWhD2EeWm4KH9E/edit).

### CODE REVIEW

File: src/main/java/seedu/address/logic/messages/RedoMessages.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Container for user redo command visible messages.
  */
 public class RedoMessages extends Messages {
-    public static final String MESSAGE_REDO_SUCCESS = "Woof! Redo successfully! \uD83D\uDC36";
-    public static final String MESSAGE_REDO_FAIL = "Woof! There is no more future state to redo!";
+    public static final String MESSAGE_REDO_SUCCESS = "Woof! Redo successful! \uD83D\uDC36";
```
Comment (Joshy837) [inline src/main/java/seedu/address/logic/messages/RedoMessages.java:7]: Good improvement on grammar here!

**Review by yleeyilin — APPROVED** [review]
lgtm but need fix check style

File: src/main/java/seedu/address/logic/messages/SearchMessages.java [hunk 2]
```diff
@@ -4,9 +4,9 @@
  * Container for user search command visible messages.
  */
 public class SearchMessages extends Messages {
-    public static final String MESSAGE_SEARCH_PERSON_SUCCESS = "Woof! %1$s contacts found! \uD83D\uDC36";
+    public static final String MESSAGE_SEARCH_PERSON_SUCCESS = "Woof! %1$s contact(s) found! \uD83D\uDC36";
```
Comment (Joshy837) [inline src/main/java/seedu/address/logic/messages/SearchMessages.java:7]: Good change of separating contact & contacts

File: src/main/java/seedu/address/logic/messages/UndoMessages.java [hunk 3]
```diff
@@ -4,7 +4,7 @@
  * Container for user undo command visible messages.
  */
 public class UndoMessages extends Messages {
-    public static final String MESSAGE_UNDO_SUCCESS = "Woof! Undo successfully! \uD83D\uDC36";
-    public static final String MESSAGE_UNDO_FAIL = "Woof! There is no more previous state to undo!";
+    public static final String MESSAGE_UNDO_SUCCESS = "Woof! Undo successful! \uD83D\uDC36";
```
Comment (Joshy837) [inline src/main/java/seedu/address/logic/messages/UndoMessages.java:7]: Good grammar

**Review by Joshy837 — APPROVED** [review]
LGTM

**Pull request merged**